pkg/reconciler/common/tektoninstallerset/defaultinstallerset_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tektoninstallerset
+
+import (
+	"context"
+	"path"
+	"testing"
+
+	mf "github.com/manifestival/manifestival"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"gotest.tools/v3/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetLabels(t *testing.T) {
+	di := NewDefaultInstaller()
+
+	di.AddLabelsFromMap(map[string]string{
+		v1alpha1.CreatedByKey: "pipeline",
+	})
+
+	labels := di.GetLabels(context.Background())
+
+	assert.Equal(t, labels[v1alpha1.CreatedByKey], "pipeline")
```

Comment:
could we use the `cmp.diff` here also to compare the input map and the output map.